### PR TITLE
Query Google Cloud Bigtable with Flux

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/flux/v0.x/query-data/bigtable.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  ```js
+  import "experimental/bigtable"
+  
+  bigtable.from(url: "http://example.com/metrics")
+  ```
+---
+
+To query [Google Cloud Bigtable](https://cloud.google.com/bigtable/) with Flux:
+
+1. Import the [`experimental/bigtable` package](/flux/v0.x/stdlib/experimental/bigtable/).
```
Comment (kelseiv) [inline content/flux/v0.x/query-data/bigtable.md:27]:
```suggestion
1.  In [InfluxDB Data Explorer]((/{{< latest "influxdb" >}}/query-data/execute-queries/data-explorer/) or the [Flux REPL], import the [`experimental/bigtable` package](/flux/v0.x/stdlib/experimental/bigtable/).
```

Comment (kelseiv) [reply content/flux/v0.x/query-data/bigtable.md:27]: @sanderson would the shortcode above work for both cloud and v2.0 context?

File: content/flux/v0.x/query-data/bigtable.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+bigtable.from(
+  token: "mySuPeRseCretTokEn",
+  project: "exampleProjectID",
+  instance: "exampleInstanceID",
+  table: "example-table"
+)
+```
+
+## Results structure
+`bigtable.from()` returns a [stream of tables](/flux/v0.x/get-started/data-structure/#stream-of-tables)
```
Comment (kelseiv) [inline content/flux/v0.x/query-data/bigtable.md:48]:
```suggestion
`bigtable.from()` returns a [stream of tables](/flux/v0.x/get-started/data-model/#stream-of-tables)
```

File: content/flux/v0.x/query-data/bigtable.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+with no grouping (all rows in a single table).
+For more information about table grouping, see
+[Flux data model - Restructure data](/flux/v0.x/get-started/data-model/#restructure-data).
```
Comment (kelseiv) [inline content/flux/v0.x/query-data/bigtable.md:51]:
```suggestion
[Flux data model - Restructure data](/flux/v0.x/get-started/data-model/#restructure-tables).
```

**Review by kelseiv — APPROVED** [review]

File: content/flux/v0.x/query-data/bigtable.md (hunk 1 above)
Comment (sanderson) [reply content/flux/v0.x/query-data/bigtable.md:27]: There are a lot of different ways they could execute this query (API, data explorer, REPL, client library, etc.). That's why I don't specifically call it out here.

**Pull request merged**